torch/ao/quantization/pt2e/utils.py

```diff
@@ -168,6 +168,7 @@ def _is_conv_node(n: Node):
     return n.op == "call_function" and n.target in [
         torch.ops.aten.conv1d.default,
         torch.ops.aten.conv2d.default,
+        torch.ops.aten.conv2d.padding,
```

Comment:
```suggestion
        torch.ops.aten.conv1d.default,
        torch.ops.aten.conv1d.padding,
        torch.ops.aten.conv2d.default,
        torch.ops.aten.conv2d.padding,
```
What about conv3d?

Reply:
I added conv3d,

@cccclai might be good to abandon these changes

Reply:
Wait so this func is unused now?

Reply:
Our partner is still using pytorch but not torch ao currently, I think it's an easy fix?

Reply:
@cccclai Might as well be complete and ad all the conv types then.